packages/core/src/local-admin-api.ts

```diff
@@ -20,13 +20,30 @@ export class LocalAdminApi implements AdminApi {
     return this.nodeStatusFn()
   }
 
-  async startIndexingModels(modelsIDs: Array<StreamID>): Promise<void> {
-    await this.indexApi.indexModels(modelsIDs)
-    await this.syncApi.startModelSync(modelsIDs.map((id) => id.toString()))
+  async startIndexingModels(
+    modelsIDs: Array<StreamID>,
+    indices?: Array<ModelFieldsIndex>
+  ): Promise<void> {
+    let models = modelsIDs.map((id) => {
+      return {
+        streamID: id,
+        indices: undefined,
+      } as ModelFieldsIndex
+    })
+    if (indices) {
+      models = models.concat(indices)
```

Comment:
this doesn't seem right?  This will just add the array of objects with both streamID and indices to the end of the array after the objects that just have streamID and an undefined indices field.  Presumably we actually want to merge and dedupe these somehow?

Reply:
They do get deduped eventually, and likely only one parameter would be passed here.